features/cli/command_line_help.feature

```diff
@@ -2,7 +2,7 @@ Feature: Command Line Help
 
   Scenario: Asking the command line tool for help
     When I run `bbc-a11y --help`
-    Then it should pass with:
+    Then it should contain with whitespace removed:
```

Comment:
I like what you've done here - but would re-word it slightly. Perhaps

```suggestion
    Then (ignoring whitespace) it should pass with:
```

Reply:
Oh my gosh that's so much better. I had a very hard time thinking of a good way to phrase this.